golang/Makefile

```diff
@@ -27,6 +27,7 @@ clean:
 	echo Cleaning build artifacts...
 	go clean
 	rm -rf $(BUILD)
+	rm -rf examples/thrift/gen-go
```

Comment:
You can put it into a generated folder if you update the Makefile GOPATH line to look in that folder.  This is how we handle the Godeps/_workspace for example.